var/spack/repos/builtin/packages/openmpi/package.py

```diff
@@ -266,6 +266,8 @@ class Openmpi(AutotoolsPackage):
 
     depends_on('pkgconfig', type='build')
 
+    depends_on('libevent@2.0:')
```

Comment:
We can also make this conditional on `@4:` since earlier releases just used the bundled libevent.
Not sure how many people rely on them, I did not test the libevent-Open MPI compatibility matrix for those `version`s